### ISSUE TITLE
dimmer only?

### ISSUE THREAD

**Issue by hepcat72** [castanhinha1/nodered-lutron-caseta#1]
I presume this repo was created for [npm instance of nodered-lutron-caseta](https://flows.nodered.org/node/nodered-lutron-caseta)...

I have a dumb question, because I know nothing about caseta.  I have a lutron dimmer switch.  Does it have a caseta hub inside it?  I.e. Can I use this node to interact with my lutron dimmer?

**Comment by ashokaiyar** [comment]
> Does it have a caseta hub inside it?

No. You can read more about Caséta [here](https://www.casetawireless.com/us/en).
